src/PlatformSockets.c

```diff
@@ -10,14 +10,15 @@ void addrToUrlSafeString(struct sockaddr_storage* addr, char* string)
 
     if (addr->ss_family == AF_INET6) {
 #ifdef __vita__
-        LC_ASSERT(FALSE);
+        struct sockaddr_in* sin = (struct sockaddr_in*)addr;
+        inet_ntop(addr->ss_family, &sin->sin_addr, addrstr, sizeof(addrstr));
```

Comment:
actually these defines are wrong.
in my test, they cause compile error without ifdef macro

```
/home/d3m3vilurr/works/vita/vita-moonlight/third_party/moonlight-common-c/src/PlatformSockets.c:13:41: error: 'struct sockaddr_in' has no member named 'sin6_addr'; did you mean 'sin_addr'?
         inet_ntop(addr->ss_family, &sin6->sin6_addr, addrstr, sizeof(addrstr));
```

so i'll fix them to right way at tonight.